classes/template/PKPTemplateManager.inc.php

```diff
@@ -1186,6 +1187,28 @@ function display($template = null, $cache_id = null, $compile_id = null, $parent
 		parent::display($template, $cache_id, $compile_id, $parent);
 	}
 
+	/**
+	 * DO NOT USE. Assign the state data to the template
+	 *
+	 * This method is a temporary workaround and should not be used
+	 * to pass state to the template. It is only used during submission
+	 * as a temporary method to initialize a Vue.js component inside
+	 * of a tab template.
+	 *
+	 * This can lead to a separate Vue instance embedded inside of the
+	 * main app's Vue instance. The instances can not talk to each other.
+	 *
+	 * This should be removed when the submission wizard is updated to
+	 * make use of the new forms powered by Vue.js.
+	 *
+	 * @deprecated 3.3
+	 * @param [type] $state
+	 * @return void
+	 */
+	function _assignState($state) {
```

Comment:
Make `private`?

Reply:
I don't think I can, since it is called from `PKPSubmissionSubmitStep2Form`, can I?

Reply:
Then it shouldn't be prefixed with a `_`. That's an old PHP convention from before `private`/`protected` were introduced.